autopilot/autopilot.go

```diff
@@ -166,13 +167,43 @@ func (ap *Autopilot) Handler() http.Handler {
 	return jape.Mux(map[string]jape.Handler{
 		"GET    /config":        ap.configHandlerGET,
 		"PUT    /config":        ap.configHandlerPUT,
+		"POST   /config":        ap.configHandlerPOST,
```

Comment:
The route is a little confusing. I think `/config/evaluate` would be better.

Reply:
Why do you think so? Imo it was kind of nice that we can use the same path for getting, settings and evaluating.

Reply:
Looking at the route `POST /config` I would not expect it to evaluate the config I send it but instead create the configuration. I think it's also a little awkward for a route to handle multiple response types, I'd expect the type returned by `GET` to be identical to what I `POST` (or `PUT`).  I don't feel strongly about it so if you want to keep it that is fine.